client/src/pages/Dashboard/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import axios from 'axios';
+
+import { Typography, Button, Grid, Box } from '@material-ui/core';
+
+import SurveyCard from './components/SurveyCard';
+import AllSurveysTable from './components/AllSurveysTable';
+
+const Dashboard = () => {
+  const [activeSurveys, setActiveSurveys] = useState(true);
```

Comment:
everything should be in redux rather than useState pls

Reply:
done